rj_gameplay/rj_gameplay/skill/capture.py

```diff
@@ -30,6 +30,10 @@ def __init__(self):
         self.root = self.capture_behavior
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot: rc.Robot) -> None:
-        self.root.tick_once(robot)
-        # TODO: change so this properly returns the actions intent messages
+    def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> None:
+    	self.robot = robot
+        self.root.tick_once(self.robot, world_state)
+
+        
+    def is_done(self, world_state) -> bool:
+    	self.capture.is_done(world_state)
```

Comment:
This should return a boolean, as per the method header. 

Again see skill/move.py in gameplay_node_update: https://github.com/RoboJackets/robocup-software/blob/gameplay_node_update/rj_gameplay/rj_gameplay/skill/move.py